client/webserver/locales/de-de.go

```diff
@@ -54,14 +65,25 @@ var DeDE = map[string]*intl.Translation{
 	"epoch":                          {T: "Epoche"},
 	"price":                          {T: "Preis"},
 	"volume":                         {T: "Volumen"},
+	"volume_24":                      {T: "24 Std. Volumen"},
+	"High":                           {T: "Hoch"},
+	"Low":                            {T: "Tief"},
 	"buys":                           {T: "Käufe"},
 	"sells":                          {T: "Verkäufe"},
 	"Buy Orders":                     {T: "Kaufaufträge"},
 	"Quantity":                       {T: "Menge"},
 	"Rate":                           {T: "Kurs"},
 	"Limit Order":                    {T: "Limit-Auftrag"},
 	"Market Order":                   {T: "Markt-Auftrag"},
-	"reg_status_msg":                 {T: `Um bei <span id="regStatusDex" class="text-break"></span> handeln zu können muss die Zahlung der Anmeldegebühr <span id="confReq"></span> bestätigt werden.`},
+	"create_account_to_trade":        {T: "Konto zum Handeln eröffnen"},
+	"need_to_register_msg":           {T: `Du musst ein Konto auf <span id=„unregisteredDex“></span> erstellen, um zu handeln.`},
+	"Create Account":                 {T: "Konto eröffnen"},
+	"reg_status_msg":                 {T: `Um bei <span id="regStatusDex" class="text-break"></span> handeln zu können muss die Zahlung der Kaution <span id="confReq"></span> bestätigt werden.`},
+	"posting_bonds_shortly":          {T: "Kaution wird gestellt..."},
+	"bond_creation_pending_msg":      {T: `Um auf <span id=„postingBondsDex“ class=„text-break“></span> zu handeln, werden in Kürze deine Kaution(en) hinterlegt.`},
```

Comment:
```suggestion
	"bond_creation_pending_msg":      {T: `Um auf <span id="postingBondsDex“ class="text-break“></span> zu handeln, werden in Kürze deine Kaution(en) hinterlegt.`},
```

Here and in a few other places, I think the quotes on the bottom are messing something up.

Reply:
Thank you for pointing this out. It was a typical german auto-correction problem. Fixed now.